wordpress-mobile/WordPress-iOS
# Testing: Add a basic publish flow test

Part of https://github.com/wordpress-mobile/WordPress-iOS/issues/11013

This PR implements a UI test for a basic publish flow:
* Create a new post.
* Add a title and post content (random text).
* Upload an image to the post.
* Add a tag, category, and featured image.
* Publish the post.

To test:

- Start the simulator with the language set to English. (Making all the tests work in any language is a followup task.)
- Make sure you have the latest mobile secrets for the iOS app.
- Run the Editor Tests in the WordPressUITests target on both iPad and iPhone.
